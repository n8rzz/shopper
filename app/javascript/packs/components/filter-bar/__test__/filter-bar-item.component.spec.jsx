import React from 'react';
import {shallow} from 'enzyme';

import {FilterBarItem} from '../filter-bar-item.component';

const filterBarItemPropsMock = {
    text: 'Alpha',
    url: '/items',
};

describe('FilterBarItem', () => {
    test('renders correctly with valid props', () => {
        const component = shallow(<FilterBarItem
            text={filterBarItemPropsMock.text}
            url={filterBarItemPropsMock.url}
        />);

        expect(component).toMatchSnapshot();
    });
});
