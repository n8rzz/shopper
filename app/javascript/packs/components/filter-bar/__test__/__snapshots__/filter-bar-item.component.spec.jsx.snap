// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterBarItem renders correctly with valid props 1`] = `
<li>
  <a
    className="filterBar-link"
    href="/items"
  >
    Alpha
  </a>
</li>
`;
